src/ssb.js

```diff
@@ -58,6 +59,13 @@ const rawConnect = () =>
         if (err) {
           reject(err);
         } else {
+          // HACK: SSB-Tangle isn't available in Patchwork, but we want that
+          // compatibility. This code automatically injects SSB-Tangle into our
+          // stack so that we don't get weird errors when using Patchwork.
+          //
+          // See: https://github.com/fraction/oasis/issues/21
+          api.tangle = ssbTangle.init(api);
```

Comment:
im not super familiar with how init and everything work, but maybe we should check if `api.tangle` is null before assigning it?

Reply:
Sure! This implementation overwrites the existing `tangle` if it exists, which I think is fine, but we can also check beforehand so that we aren't modifying the object unless we need to.